docs/x509.rst

```diff
@@ -431,6 +431,99 @@ X.509 Extensions
 
         Returns an instance of the extension type corresponding to the OID.
 
+.. class:: KeyUsage
+
+    .. versionadded:: 0.9
+
+    The key usage extension defines the purpose of the key contained in the
+    certificate.  The usage restriction might be employed when a key that could
+    be used for more than one operation is to be restricted. It corresponds to
+    :data:`OID_KEY_USAGE`.
+
```

Comment:
I was discussing this with @jmvrbanac today. Iteration of the properties is certainly doable but I've struggled to come up with a use case where it would actually be helpful?